moped-editor/src/views/projects/projectView/ProjectComponents/ProjectComponentsList.js

```diff
@@ -88,13 +93,28 @@ const ProjectComponentsList = ({
                     fullWidth
                     size="small"
                     color="primary"
-                    startIcon={<EditOutlined />}
-                    onClick={() => onStartEditingComponent(component)}
+                    startIcon={<ListIcon />}
+                    onClick={onEditAttributes}
                   >
-                    Edit
+                    Attributes
```

Comment:
Ah—i forgot that Amenity offered a better term for this. Can you label this "Details" instead of "Attributes?

Reply:
✅ 🙏